tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -38,7 +38,7 @@ final class ClosureLinkerBackend(
     outputMode: OutputMode,
```

Comment:
Also, if we currently can't have ES5, we should just not have the option and add it later...